### ISSUE TITLE
Unable to install with Quicklisp on Windows 10, SBCL 2.0

### ISSUE THREAD

**Issue by iantor** [rabbibotton/clog#65]
I saw in the manual that Clog is periodically tested on Windows.
I currently have SBCL 2.0, the only official binary for Windows, used with Emacs 27.2 native for Windows and Slime, on Windows 10.
I followed the instructions to install with Quicklisp, but ran into the following error.

[package cl+ssl]
; 
; compilation unit aborted
;   caught 1 fatal ERROR condition
; Evaluation aborted on #<CFFI:LOAD-FOREIGN-LIBRARY-ERROR "Unable to load any of the alternatives:~%   ~S" {100427AA73}>.

My plan now, rather than trying to compile a more up-to-date SBCL for Windows, is to use Docker, and pick up a standard Docker image for SBCL, and use it locally with Docker Desktop, but eventually deploy on  a VPS with the same Docker image.
Still curious though if there is a quick fix that would allow me to play with this using Windows SBCL.

**Comment by rabbibotton** [comment]
The issues is not with CLOG but with the cl+ssl package. Maybe the maintainer of that has more insight on that running on windows 10. I haven't tried lately but will.

**Comment by iantor** [comment]
Thanks.  Here is some relevant info that I found: https://www.reddit.com/r/lisp/comments/a8dzhi/windows_10_version_of_openssl_for_clssl/

**Comment by iantor** [comment]
I was able to install Clog on Windows 10 with sbcl 2.0.0.  I first installed the 64-bit light version of openssl from [Shining Light](http://slproweb.com/products/Win32OpenSSL.html).
I then also installed the 64-bit sqlite dll  from the [home page](https://www.sqlite.org/index.html) as its absence also generated a Quickload error.  I put that into \windows\system32 . Finally the Quickload installation worked.
Will test Clog within a couple of days.
